gitlab/resource_gitlab_project.go

```diff
@@ -89,6 +89,30 @@ func resourceGitlabProject() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"shared_with_groups": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"group_id": {
+							Type:     schema.TypeInt,
+							Required: true,
+						},
+						"group_access": {
```

Comment:
Why not `group_access_level` to match the API?

https://docs.gitlab.com/ee/api/groups.html#details-of-a-group